MANIFEST.in

```
@@ -6,3 +6,4 @@ include pyproject.toml
 include requirements/requirements.common.txt
 graft galaxy_ng/app/webserver_snippets
 graft galaxy_ng/app/static
+recursive-include locale *.po
```

After you move the `locale` to `galaxy_ng/locale`

```suggestion
graft galaxy_ng/locale
```

`graft` takes the whole folder and adds to the distribution binaries.